### PR TITLE
Removing Buchi from CODEOWNERS

### PR DESCRIPTION
**Pull request by buchi-busireddy**
## Description
Removing Buchi from CODEOWNERS

### Testing
No testing done since there are no code changes.

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by JBAhire — APPROVED** [review]
Hey @buchi-busireddy @aaron-steinfeld , as we are doing this across all repos anyways, can we create a CI team and query-service team and add them as codeowners instead? Because otherwise, this will be the problem everytime someone leaves. 

Just a suggestion because, Buchi has to raise PRs in many repos yet so let's make efficient use of this effort. 😅

**Comment by aaron-steinfeld** [comment]
Yeah, code owners should generally reference teams rather than people. I'll create three teams with at least 2 people IST and 2 US:
ci-owners, config-services-owners and query-layer-owners (just checked and this last one already exists, will add more folks to it)

**Comment by aaron-steinfeld** [comment]
> Yeah, code owners should generally reference teams rather than people. I'll create three teams with at least 2 people IST and 2 US:
> ci-owners, config-services-owners and query-layer-owners (just checked and this last one already exists, will add more folks to it)

Done. I kind of guessed a bit at membership, no slights intended - feel free to adjust as long as we keep at least 2 folks on each side of the globe :)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # global
-* @buchi-busireddy @avinashkolluru @tim-mwangi @aaron-steinfeld
+* @avinashkolluru @tim-mwangi @aaron-steinfeld
```
Comment (aaron-steinfeld) [inline .github/CODEOWNERS:4]:
```suggestion
* @hypertrace/config-services-owners
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 # GH action
-.github/ @aaron-steinfeld @jbahire @kotharironak @buchi-busireddy
+.github/ @aaron-steinfeld @jbahire @kotharironak
```
Comment (aaron-steinfeld) [inline .github/CODEOWNERS:7]:
```suggestion
.github/ @hypertrace/ci-owners
```

**Review by aaron-steinfeld — COMMENTED** [review]
sorry, hijacked the PR via suggestions so I didn't have to create my own 😉

**Review by JBAhire — APPROVED** [review]

**Comment by JBAhire** [comment]
Nice!

**Comment by aaron-steinfeld** [comment]
> @buchi-busireddy , use same everywhere now 😂
> 
> Just 38 more to go.

We should probably not make buchi do all of them, seems a bit unfair :)

**Comment by aaron-steinfeld** [comment]
Oh I forgot, teams also need to be added explicitly to each repo for them to be usable as code owners. That's always annoyed me...

**Comment by JBAhire** [comment]
> Oh I forgot, teams also need to be added explicitly to each repo for them to be usable as code owners. That's always annoyed me...

Yeah, this is one time hassle now but will be helpful moving forward, as we grow. :)

**Pull request merged**